packages/widget/src/pages/SwapPage/SwapPage.tsx

```diff
@@ -304,6 +305,7 @@ export const SwapPage = () => {
           }
           setChainAddresses({});
           setCurrentPage(Routes.SwapExecutionPage);
+          setUser({ username: sourceAccount?.address });
           setSwapExecutionState();
```

Comment:
lets also capture the route here

Reply:
how should i capture the route?

Reply:
not blocking the merge, we can use this https://docs.sentry.io/platforms/javascript/enriching-events/context/

Reply:
hmm, ok I'll test tags and contexts in a separate PR to decide which one to use

https://docs.sentry.io/platforms/javascript/enriching-events/tags/
https://docs.sentry.io/platforms/javascript/enriching-events/context/